app/src/main/res/values/strings.xml

```diff
@@ -1,34 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <!-- NavigationDrawer -->
```

Comment:
Could you also organize these sections to be in alphabetical order per the third point in #3384? We do that elsewhere, so I think it'd make sense to also do it here.

Reply:
Okay sure @BenHenning I change this.

Reply:
Done.